src/randoop/NoExceptionCheck.java

```diff
@@ -12,9 +12,11 @@
   
   // Indicates which statement is expected to return normally. 
   private final int statementIdx;
+  private String exceptionName;
   
-  public NoExceptionCheck(int statementIdx) {
+  public NoExceptionCheck(int statementIdx, String exceptionName) {
```

Comment:
I edited the class comment to say

> This check represents the fact that a statement should not throw an exception.
> It is used in an error-revealing test to indicate that an exception that is
> considered to be an error (e.g., not "expected" or "invalid") was thrown by 
> the statement during test generation. Only a comment is included when the test is
> output noting the occurrence of the exception during test generation, though
> the statement is expected to throw the method when the error-revealing test 
> is run.

I suppose I did change the intent of the class by adding the exception name, but my feeling is that if the point of an Error-Revealing test is to *reveal* then we should say what exception was observed during test generation.